_data/simple-icons.json

```
@@ -2523,6 +2523,11 @@
             "hex": "FFFFFF",
             "source": "https://clarivate.com"
         },
+        {
+            "title": "Clerk",
+            "hex": "000000",
+            "source": "https://clerk.com/_next/image?url=%2Fimages%2Ffooter-logo.svg&w=32&q=75"
```

```suggestion
            "hex": "6C47FF",
            "source": "https://clerk.com"
```

Hex code is available as the primary colour of the buttons etc on the site.
URL-wise we can get away with just the homepage.